src/main/kotlin/gg/flyte/christmas/minigame/games/KingHill.kt

```diff
@@ -13,30 +13,48 @@ import gg.flyte.twilight.extension.playSound
 import gg.flyte.twilight.scheduler.delay
 import gg.flyte.twilight.scheduler.repeatingTask
 import gg.flyte.twilight.time.TimeUnit
+import net.kyori.adventure.bossbar.BossBar
 import net.kyori.adventure.text.Component
-import org.bukkit.Bukkit
-import org.bukkit.GameMode
-import org.bukkit.Material
-import org.bukkit.Sound
+import org.bukkit.*
+import org.bukkit.attribute.Attribute
+import org.bukkit.attribute.AttributeModifier
 import org.bukkit.enchantments.Enchantment
 import org.bukkit.entity.ItemDisplay
 import org.bukkit.entity.Player
 import org.bukkit.event.EventPriority
 import org.bukkit.event.entity.EntityDamageByEntityEvent
 import org.bukkit.event.entity.EntityDamageEvent
 import org.bukkit.event.player.PlayerMoveEvent
+import org.bukkit.event.player.PlayerToggleFlightEvent
+import org.bukkit.inventory.EquipmentSlotGroup
 import org.bukkit.inventory.ItemStack
 import org.bukkit.inventory.meta.SkullMeta
+import org.bukkit.potion.PotionEffect
+import org.bukkit.potion.PotionEffectType
+import org.bukkit.util.Vector
 import java.time.Duration
 import java.util.*
+import kotlin.math.floor
 
 class KingHill : EventMiniGame(GameConfig.KING_OF_THE_HILL) {
     private var hillRegion = MapRegion(MapSinglePoint(824, 85, 633), MapSinglePoint(830, 88, 627))
     private var pvpEnabled = false
-    private var gameTime = 90
-    private val respawnBelow = 71
+    private var gameTime = 150
+    private val respawnBelow = 60
```

Comment:
Yeah, I believe in my commit messages I said the following:
- I decreased the `respawnBelow` value as this gives double jumps more use (save yourself from falling into the void, the previous height was too high to react fast enough).
- I increased the game time as 90 seconds felt really short during my testing with @rolyPolyVole, and with even more players I don't think it will be long enough to have an interesting game. You can change it back if you like but I think this is the correct decision.